WebApp/static/DIRAC/ConfigurationManager/classes/ConfigurationManager.js

```diff
@@ -251,16 +251,16 @@ Ext.define('DIRAC.ConfigurationManager.classes.ConfigurationManager', {
 
               case "setOptionValue" :
                 var oNode = me.treeStore.getNodeById(oResponse.parentNodeId);
-                oNode.raw.csValue = oResponse.value;
+                oNode.get('csValue') = oResponse.value;
```

Comment:
I don't know much about js but this statement doesn't seem correct to me... can you do assignments like this?

Reply:
rigt! Setter method must be used. I corrected